lambda/main.py

```diff
@@ -45,10 +45,10 @@ def get_instance_id(event):
         LOGGER.error(err)
         return False
 
-def get_pem():
+def get_pem(cipherfile):
     """Decrypt the Ciphertext Blob to get USERNAME's pem file"""
     try:
-        with open('encrypted_pem.txt', 'r') as encrypted_pem:
+        with open('cipherfile', 'r') as encrypted_pem:
```

Comment:
cipherfile is a variable now. it shouldn't be surrounded by any quotes otherwise it will look for a file named "cipherfile".